sklearn/decomposition/factor_analysis.py

```diff
@@ -61,7 +61,7 @@ class FactorAnalysis(BaseEstimator, TransformerMixin):
         If None, n_components is set to the number of features.
 
     tol : float
-        Stopping tolerance for EM algorithm.
+        Stopping tolerance for SVD based approach.
```

Comment:
It's definitely not a stopping tolerance for the svd. It's tolerance of the log likelihood that is being maximised by em

Reply:
because SVD based approach also   iterate, i think it need this stop tolerance

Reply:
"Stopping tolerance for log-likelihood increase"?

Reply:
i think that's fine. should i change it ?

Reply:
Please do if you think it's good